_posts/2022-10-28-multiple-data-source.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Launch Highlight: Multi Data Source"
+authors:
+  - tyarong
+  - zengyan
+  - kroosh
```

Comment:
would you check add add md and profile photo for new authors

https://github.com/opensearch-project/project-website/tree/main/_authors